Cargo.toml

```diff
@@ -23,12 +23,12 @@ read_buf = ["rustls/read_buf"]
 
 [dependencies]
 # Keep in sync with RUSTLS_CRATE_VERSION in build.rs
-rustls = { version = "=0.22.0-alpha.4", features = [ "ring" ]}
-rustls-webpki = "0.102.0-alpha.6"
-pki-types = { package = "rustls-pki-types", version = "0.2.1", features = ["std"] }
+rustls = { version = "=0.22.0-alpha.6", features = [ "ring" ]}
+pki-types = { package = "rustls-pki-types", version = "0.2.3", features = ["std"] }
```

Comment:
This changes to 0.2.3 instead of staying at 0.2.2 because we're using the `ServerName::to_str` method that's only available in 0.2.3+